docs/src/howto/reduction.md

```diff
@@ -31,10 +31,10 @@ julia> OK = maximal_order(K);
 julia> p = 29;
 
 julia> frakp = prime_decomposition(OK, p)[1][1]
-<29, z_7 + 22>
-Norm: 29
-Minimum: 29
-two normal wrt: 29
+Ideal of order of cyclotomic field of order 7
+  with 29-normal generators [29, z_7 + 22]
+  of norm 29
+  of minimum 29
```

Comment:
And this printed as
```
Ideal
  of order of cyclotomic field of order 7
with 29-normal generators [29, z_7 + 22]
  of norm 29
  of minimum 29
```

Reply:
This make it look like the second `of` are properties of the `with 29-normal ...` line.

Reply:
Yes that's my bad, I misunderstood it. Then why not keeping (non-indented)  `Norm: 29` and `Minimum: 29` as it was before ?

Reply:
I find the rule of what is intended and what not quite arbitrary. So I intended to indent everything consistently.

Indentation should help parse things. But if indentation is not consistent, this does not help. For example printing many ideals would give something like
```
Ideal
  of order of cyclotomic field of order 7
with 29-normal generators [29, z_7 + 22]
  of norm 29
  of minimum 29
Ideal
  of order of cyclotomic field of order 7
with 29-normal generators [29, z_7 + 22]
  of norm 29
  of minimum 29
Ideal
  of order of cyclotomic field of order 7
with 29-normal generators [29, z_7 + 22]
  of norm 29
  of minimum 29
```
It is now difficult to see where one thing starts/ends.